routes/web.php

```diff
@@ -13,34 +13,75 @@
 
 Route::get('/', function () {
   return view('patient');
-});
+})->name('patient');
 
+Auth::routes();
 
-Route::get('/myworkflows', function () {
-  return view('myworkflows');
-});
+Route::get('/emailverify/{token}', 'Auth\RegisterController@verifyUser')->name('emailverification');
 
-Auth::routes();
+Route::get('/usersverification', 'UsersVerificationController@index')->name('usersverification.index');
+
+Route::get('/usersverification/download/{id}', 'UsersVerificationController@download')->name('usersverification.download');
+Route::post('/usersverification/accept', 'UsersVerificationController@accept')->name('usersverification.accept');
```

Comment:
It's best if you split the route file up in multiple files such that each file belongs to a single entity.